tests/models/test_anthropic.py

```diff
@@ -66,8 +67,8 @@ def test_init():
 
 @dataclass
 class MockAnthropic:
-    messages_: AnthropicMessage | list[AnthropicMessage] | None = None
-    stream: list[RawMessageStreamEvent] | list[list[RawMessageStreamEvent]] | None = None
+    messages_: AnthropicMessage | Exception | list[AnthropicMessage | Exception] | None = None
+    stream: list[RawMessageStreamEvent | Exception] | list[list[RawMessageStreamEvent | Exception]] | None = None
```

Comment:
Could make an alias for these things to make it slightly easier to visually consume:
```suggestion
MockAnthropicMessage = AnthropicMessage | Exception
MockRawMessageStreamEvent = RawMessageStreamEvent | Exception
...

    messages_: MockAnthropicMessage | list[MockAnthropicMessage] | None = None
    stream: list[MockRawMessageStreamEvent] | list[list[MockRawMessageStreamEvent]] | None = None
```
Or could leave as is.

Reply:
Yes good idea, happy to